spec/spec_helper.rb

```diff
@@ -131,6 +131,8 @@
   end
 end
 
+RSpec::Matchers.define_negated_matcher :not_change, :change
```

Comment:
Did `.not_to change{}` not work for some reason? It's not really more grammatical this way or anything?